pkg/asset/machines/worker.go

```diff
@@ -158,6 +162,15 @@ func defaultVSphereMachinePoolPlatform() vspheretypes.MachinePool {
 	}
 }
 
+func defaultPowerVSMachinePoolPlatform() powervstypes.MachinePool {
+	return powervstypes.MachinePool{
+		Memory:     "32",
+		Processors: "0.5",
+		ProcType:   "shared",
```

Comment:
Probably want to update this to the enum here?

Reply:
ah, yah thanks